model/src/main/java/org/projectnessie/api/v2/params/GetReferenceParams.java

```diff
@@ -31,8 +29,21 @@
 public class GetReferenceParams {
 
   @Parameter(
-      description = REF_NAME_DESCRIPTION,
-      examples = {@ExampleObject(ref = "ref")})
+      description =
+          "A reference to a particular version of the contents tree (a point in history).\n"
```

Comment:
Should we maybe move the text `ApiDoc` near `REF_PARAMETER_DESCRIPTION` as `REF_NAME_PARAMETER_DESCRIPTION`?

Reply:
moved